src/shipit_code_coverage/tests/test_artifacts.py

```diff
@@ -59,3 +60,25 @@ def add_dir(files):
 
     with pytest.raises(Exception, message='suite and chunk can\'t both have a value'):
         a.get(chunk='xpcshell-7', suite='mochitest')
+
+
+@mock.patch('shipit_code_coverage.taskcluster.get_task_artifacts')
+@mock.patch('shipit_code_coverage.taskcluster.download_artifact')
+def test_download(mocked_download_artifact, mocked_get_task, LINUX_TEST_TASK_ARTIFACTS):
+    a = ArtifactsHandler([], [], parent_dir='')
```

Comment:
It should work if you don't define parent_dir too, can you make sure?

Reply:
If I don't define parent_dir at all, it gives me an assertation error, showing that the mock.call() is '' and not what expected.

Reply:
Actually, it did work :)